python/ql/src/semmle/python/essa/Essa.qll

```diff
@@ -609,6 +609,11 @@ class MultiAssignmentDefinition extends EssaNodeDefinition {
         )
     }
 
+    /**
+     * Holds if `this` has index `index` in `lhs`.
+     *
+     * For example, for `a,b = t` the call `b.indexOf(1, t)` holds.
```

Comment:
There's a category error here: the `b` and `t` in the second code snippet are not the same as in the first snippet.